service/tcp.go

```diff
@@ -62,12 +62,13 @@ func remoteIP(conn net.Conn) netip.Addr {
 	return netip.Addr{}
 }
 
-// Wrapper for logger.Debugf during TCP access key searches.
-func debugTCP(cipherID, template string, val interface{}) {
+// Wrapper for slog.Debug during TCP access key searches.
+func debugTCP(template string, cipherID string, args ...any) {
 	// This is an optimization to reduce unnecessary allocations due to an interaction
-	// between Go's inlining/escape analysis and varargs functions like logger.Debugf.
-	if logger.IsEnabledFor(logging.DEBUG) {
-		logger.Debugf("TCP(%s): "+template, cipherID, val)
+	// between Go's inlining/escape analysis and varargs functions like slog.Debug.
+	if slog.Default().Enabled(context.TODO(), slog.LevelDebug) {
+		args = append(args, slog.String("ID", cipherID))
+		slog.Debug(fmt.Sprintf("TCP: %s", template), args...)
```

Comment:
FYI. I guess it doesn't matter for debug, but `LogAttrs` is the most efficient way to log: https://pkg.go.dev/golang.org/x/exp/slog#hdr-Attrs_and_Values

Reply:
Good point. Done, even if it's just for debug.